tools/docker/scripts/install_python.sh

```diff
@@ -20,19 +22,4 @@ rm -rf /var/lib/apt/lists/*
 
```

Comment:
The `build-essential` should not be needed any more, and the `python-dev*` packages can probably go as well.

Reply:
Possibly, but not sure: we're installing `numpy` via pip which may or may not need to build something (depending on whether a wheel is available or not for that distro+version of python).
I'd keep it for now in case they're dropping the wheels for Python 2 at some point.